packages/app/src/components/Randomizer/Randomizer.test.js

```diff
@@ -1,16 +1,17 @@
 import React from 'react'
-import { fireEvent, waitForDomChange } from '@testing-library/react'
+import { fireEvent, waitForDomChange, act } from '@testing-library/react'
```

Comment:
See other suggestions for why `wait` is necessary.
```suggestion
import { fireEvent, waitForDomChange, act, wait } from '@testing-library/react'
```